.travis.yml

```diff
@@ -25,6 +25,11 @@ script:
   - flake8 *.py tests ga4gh scripts --exclude=ez_setup.py
   - nosetests --with-coverage --cover-package ga4gh
               --cover-inclusive --cover-min-percentage 85
-              --cover-branches --cover-erase
+              --cover-branches --cover-erase -a '!travis_only'
```

Comment:
This means "do not run tests tagged with the attribute `travis_only`".  We do not want to run tests with installed executables in development, since they might be (probably are) from a different code base that the one that one is changing.